### DISCUSSION TITLE
Enyaq

### DISCUSSION THREAD

**Discussion by hessenkletterer** [tillsteinbach/WeConnect-mqtt#275] [Ideas]
Is it also possible to use your script for an Skoda Enyaq? I can also give qou access to my account at MySkoda to develop this connection. Thanks to get some more informations.

**Comment by joe2470** [comment]
Hi.
I would be also highly interested in a connection to Skoda MySkoda App to connec to my Enyaq which will be delivered soon. 
Is there any progress in that topic or further support needed?
Thank you!

**Comment by tillsteinbach** [comment]
I‘ll get my Enyaq in a few months. Then I can have a look.
